### DISCUSSION TITLE
Stand alone GB/GBC

### DISCUSSION THREAD

**Discussion by ZacharyRanes** [eduardofilo/RG350_adam_image#235] [Ideas & Requests]
Is there any chance in a future release for there to be a stand alone GB/GBC emulator like [gambatte-dms]( https://github.com/bardeci/gambatte-dms) to be included in the image?

**Comment by eduardofilo** [comment]
The design principle of the image is rather the opposite, i.e. to phase out standalone emulators as the corresponding RA cores improve.
You can always complete the image with the emulators you are interested in. Just copy them to the `apps` directory of the external card (the internal card is also suitable, but in general I recommend adding content to the external card to make it easier to switch between versions of the image). Then you only have to add the path to the OPK of the emulator in the SimpleMenu configuration file of the section containing the system to emulate.
